.github/workflows/test.yml

```
@@ -0,0 +1,174 @@
+        fi
+    
+    - name: Comment PR with coverage
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v7
+      continue-on-error: true
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
+          const fs = require('fs');
+          let coverageComment = '## 📊 Test Coverage Report\n\n';
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)